src/provision/freedom-module.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "provision",
+  "description": "Provisions VMs on Digital Ocean",
+  "app": {
+    "script": [
+      "/forge-min/forge.min.js",
```

Comment:
do scripts with an absolute path work everywhere? we've normally been using relative paths.